docs/faq.md

```diff
@@ -158,14 +158,29 @@ docker exec -t pmm-server bash -c  "grafana-cli --homepath /usr/share/grafana a
 
 If you're deploying through Docker you can change the default password for an admin user after starting the Docker container as follows:
 
-    * For PMM versions 2.27.0 and later:
+* For PMM versions 2.27.0 and later:
 
-    ```sh
-    docker exec -t pmm-server change-admin-password <new_password>
-    ```
+```sh
+docker exec -t pmm-server change-admin-password <new_password>
+```
+
+* For PMM versions prior to 2.27.0:
+
+```sh
+docker exec -t pmm-server bash -c 'grafana-cli --homepath /usr/share/grafana --configOverrides cfg:default.paths.data=/srv/grafana admin reset-admin-password newpass'
```

Comment:
is this a change from other PR? not this one?

Reply:
Actually, it was just an indent issue so I addressed this as a part of this PR.